package-lock.json

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
Using lock files for libraries is not an obvious choice. Please keep this PR only about dropping `gulp-util`. Unless you know the opinion of the maintainer about it, please open a second PR if you want to discuss the addition of the lock file.